### PR TITLE
Fb changes to crypt

### PR DESCRIPTION
**Pull request by mikedodge04**
Fb changes to crypt

### CODE REVIEW

**Comment by erikng** [comment]
Should we do anything about the issue I found? This will fail if targeted on 10.11 and lower machines.

**Comment by erikng** [comment]
Maybe just add a note in the readme that this is designed for 10.12 and higher?

File: cpe_crypt/README.md [hunk 1]
```diff
@@ -45,18 +49,18 @@ The profile delivers a payload for the above keys in `node['cpe_crypt']`.  The t
...
-    node.default['cpe_crypt']['package'] = {
+    node.default['cpe_crypt']['prefs']['RemovePlist'] = false
+    node.default['cpe_crypt']['prefs']['RotateUsedKey'] = false
+    node.default['cpe_crypt']['pkg'] = {
       'app' => 'crypt',
       'checksum' =>
         '1582e974820a5b27cfe462521cb0d4802319224753f3d6417ee23fff9333872a',
-      'receipt' => 'com.grahamgilbert.Crypt',
+      'receipt' => 'com.grahamgilbert.crypt',
       'version' => '3.0.0.109',
```
Comment (clburlison) [inline cpe_crypt/README.md]: I am not sure I agree with this case change. Maybe fb is rebuilding crypt? However the default receipt has a capital 'C' and `pkgutil` is case sensitive. 

```bash
$ pkgutil --pkg-info com.grahamgilbert.Crypt
package-id: com.grahamgilbert.Crypt
version: 3.0.0.109
volume: /
location: /
install-time: 1507948451

$ pkgutil --pkg-info com.grahamgilbert.crypt
No receipt for 'com.grahamgilbert.crypt' found at '/'.
```

**Comment by clburlison** [comment]
Thanks for these changes Mike. Much cleaner!

File: cpe_crypt/README.md (hunk 1 above)
Comment (erikng) [reply cpe_crypt/README.md]: I agree. I will merge this, but fix.

**Comment by mikedodge04** [comment]
Thanks, for taking care of this. I was hoping to get to the requested
changes today, but a bunch of stuff came up.

File: cpe_crypt/resources/cpe_crypt_install.rb [hunk 2]
```diff
@@ -10,17 +10,74 @@
...
+  end
+
+  def crypt_installed?
+    Chef::Log.warn('Crypt binary not installed!')
+    crypt_path =
+      '/Library/Security/SecurityAgentPlugins/Crypt.bundle/Contents/MacOS/Crypt'
+    ::File.exist?(crypt_path)
+  end
+
+  def vaildate_install
```
Comment (mikedodge04) [inline cpe_crypt/resources/cpe_crypt_install.rb]: i can't spell :'(

File: cpe_crypt/resources/cpe_crypt_install.rb [hunk 3]
```diff
@@ -10,17 +10,74 @@
...
+    execute "/usr/sbin/pkgutil --forget #{pkg['receipt']}" do
+      not_if do
+        shell_out("/usr/sbin/pkgutil --pkg-info #{pkg['receipt']}").error?
+      end
+      action :run
+    end
+  end
+
+  def crypt_installed?
+    Chef::Log.warn('Crypt binary not installed!')
```
Comment (mikedodge04) [inline cpe_crypt/resources/cpe_crypt_install.rb]: remove

**Pull request merged**